spec/chromium-spec.js

```diff
@@ -937,6 +937,20 @@ describe('chromium feature', () => {
   })
 
   describe('storage', () => {
+    describe('DOM storage quota override', () => {
+      ['localStorage', 'sessionStorage'].forEach((storageName) => {
+        it(`allows saving at least 50MiB in ${storageName}`, () => {
+          const storage = window[storageName]
+          const testKeyName = '_electronDOMStorageQuotaOverrideTest'
+          // 25 * 2^20 UTF-16 characters will require 50MiB
+          const arraySize = 25 * Math.pow(2, 20)
+          storage[testKeyName] = new Array(arraySize).fill('X').join('')
+          expect(storage[testKeyName]).to.have.lengthOf(arraySize)
```

Comment:
@deepak1556 I think this test is actually not working correctly. It appears to pass because no exceptions are thrown, but the data isn't really being persisted, and this can be seen by separating the read and write operations by a `setTimeout`, page refresh, etc. See https://github.com/electron/electron/issues/13465#issuecomment-494983533 . Any idea how we can fix this?